htdocs/projet/tasks/task.php

```diff
@@ -576,28 +576,28 @@
 				// Modify
 				if ($user->rights->projet->creer)
 				{
-					print '<a class="butAction" href="'.$_SERVER['PHP_SELF'].'?id='.$object->id.'&amp;action=edit&amp;withproject='.$withproject.'">'.$langs->trans('Modify').'</a>';
+					print '<div class="inline-block divButAction"><a class="butAction" href="'.$_SERVER['PHP_SELF'].'?id='.$object->id.'&amp;action=edit&amp;withproject='.$withproject.'">'.$langs->trans('Modify').'</a></div>';
```

Comment:
According to the template myobject_card.php in v10, this div seems not required. Do you experience a need for this div ?

Reply:
It's not align in Oblyon. Possible to include in v10 ?

Reply:
The <div class="inline-block divButAction"> were removed in most butAction in v10. 
If there is a trouble with oblion, it means you are missing something in oblyion css.

Because butAction are inside a tabsAction you have enough tags to fix oblyon. You can try to introduce in CSS :

.tabsAction .butAction {
    display: inline-block;
}

Or, another exemple used into md theme to have buttons aligned

div.tabsAction > a.butAction, div.tabsAction > a.butActionRefused, div.tabsAction > a.butActionDelete,
div.tabsAction > span.butAction, div.tabsAction > span.butActionRefused, div.tabsAction > span.butActionDelete {
	margin-bottom: 1.4em !important;
}

Depending on oblyon css, it may need something else, but if eldy and md have buttons aligns with the css, you should be able to get it to into oblyon.

Reply:
Thanks Laurent. I close this PR